src/utils/ethereumUtils.ts

```diff
@@ -570,10 +570,6 @@ async function parseEthereumUrl(data: string) {
   let nativeAmount: any = null;
   const { nativeCurrency } = store.getState().settings;
 
-  while (store.getState().data.isLoadingAssets) {
```

Comment:
this is already handled at its call site in `deeplinks.ts` the only other instance is via QR scan which isnt available until the app is loaded already